README.md

```diff
@@ -99,27 +99,34 @@ name the -B and -T options allow a mapping file to be specified to
 rename branches and tags (respectively). The syntax of the mapping
 file is the same as for the author mapping.
 
-Content filtering
+Plugins
 -----------------
 
-hg-fast-export supports filtering the content of exported files.
-The filter is supplied to the --filter-contents option. hg-fast-export
-runs the filter for each exported file, pipes its content to the filter's
-standard input, and uses the filter's standard output in place
-of the file's original content. The prototypical use of this feature
-is to convert line endings in text files from CRLF to git's preferred LF:
+hg-fast-export supports plugins to manipulate the file data and commit
+metadata. The plugins are enabled with the --plugin option. The value
+of said option is a plugin name (by folder in the plugins directory),
+and optionally, and equals-sign followed by an initialization string. 
+
```

Comment:
I didn't see that there was a way to pass parameters to the plugin in my first reading and was going to suggest it now, nice work!